README.md

```diff
@@ -18,35 +29,59 @@ It also integrates with Windows Explorer and Microsoft Visual Studio (2010/2012/
   </tr>
   <tr>
     <td>Requirement</td>
-    <td>MS Windows 7SP1+ <br/>MS .NET Framework 4.6.1+</td>
-    <td><a href="http://www.mono-project.com/download/">Mono 5.0+</a> <br /><small>(deprecated support due to Mono WinForms implementation)</small></td>
+    <td>MS Windows 7SP1+ <br/>MS .NET Framework 4.6.1+<br />MS VS 2017</td>
+    <td>No official support</td>
   </tr>
   <tr>
     <td>Current dev status</td>
-    <td><a href="https://ci.appveyor.com/project/gitextensions/gitextensions/branch/master"><img alt="Build status" src="https://ci.appveyor.com/api/projects/status/yo5kw7sl6da8danr/branch/master?svg=true" style="max-width:100%;"></a> <a href="https://codecov.io/gh/gitextensions/gitextensions"><img alt="codecov.io" src="https://codecov.io/gh/gitextensions/gitextensions/branch/master/graph/badge.svg" style="max-width:100%;"></a></td>
-    <td><a href="https://travis-ci.org/gitextensions/gitextensions"><img alt="Build status" src="https://travis-ci.org/gitextensions/gitextensions.svg?branch=master" style="max-width:100%;"></a></td>
```

Comment:
I suggest removing the table completely for 3.0

Reply:
I mean remove the Linux/Mac column and convert the table to text (keeping one column is OK)

Reply:
Additional: For both 2.5x and 3.0, move MS VS to a separate row/line: Development

Reply:
I actually think that having "Linux" column for v3 is beneficial to make it clear to the users